lib/datastore/query.js

```diff
@@ -109,29 +109,32 @@ Query.prototype.autoPaginate = function(autoPaginateVal) {
  *
  * @resource [Datastore Filters]{@link https://cloud.google.com/datastore/docs/concepts/queries#Datastore_Filters}
  *
- * @param {string} filter - Property + Operator (=, <, >, <=, >=).
+ * @param {string} property - The field name.
+ * @param {string=} operator - Operator (=, <, >, <=, >=). Default: `=`.
  * @param {*} value - Value to compare property to.
  * @return {module:datastore/query}
  *
  * @example
  * // List all companies named Google that have less than 400 employees.
  * var companyQuery = query
- *   .filter('name =', 'Google')
- *   .filter('size <', 400);
+ *   .filter('name', '=', 'Google')
```

Comment:
Can you squeeze in an example showing the simplest use possible, something like...

```js
 * //-
 * // List all companies that are located in California.
 * //-
 * var caliQuery = query.filter('state', 'CA');
 *
 * //-
 * // List all companies named Google that have less than 400 employees.
 * ...
```